src/NonCumulative/Soundness/Properties/LogRel.agda

```diff
@@ -0,0 +1,45 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Level hiding (_⊔_)
+open import Axiom.Extensionality.Propositional
+
+-- Properties of the gluing models for terms and types
+module NonCumulative.Soundness.Properties.LogRel (fext : Extensionality 0ℓ (suc 0ℓ)) (fext₁ : Extensionality (suc 0ℓ) (suc 0ℓ)) where
+
+open import Lib
+open import Data.Nat
+open import Data.Nat.Properties as ℕₚ
+
+open import NonCumulative.Statics.Ascribed.Misc
+open import NonCumulative.Statics.Ascribed.Presup
+open import NonCumulative.Statics.Ascribed.Properties
+open import NonCumulative.Semantics.Readback
+open import NonCumulative.Semantics.Properties.PER fext
+open import NonCumulative.Soundness.LogRel
+
+open import NonCumulative.Soundness.Properties.NoFunExt.LogRel public
+
+Glu-wellfounded-≡′ : ∀ {j i i′} → (j<i : j < i) → (j<i′ : j < i′) → 
+  (λ (univ : ∀ {l} → l < j → Ty) {A B} → Glu-wellfounded i j<i univ {A} {B}) ≡ (λ (univ : ∀ {l} → l < j → Ty) {A B} → Glu-wellfounded i′ j<i′ univ {A} {B})
+Glu-wellfounded-≡′ {j} {i} {i′} (s≤s {j} j<i) (s≤s {j} j<i′) = fext₁ λ (univ : ∀ {l} → l < j → Ty) → cong (λ 
+  (rc : ∀ {k} (k<i : k < j) (univ : ∀ {l} → l < k → Ty) {A B} → Ctx → Typ → A ≈ B ∈ PERDef.𝕌 k univ → Set) {A B} → 
+  Glu.⟦ j , rc , univ ⟧_⊢_®_) (implicit-extensionality fext λ {j′} → fext λ j′<j → Glu-wellfounded-≡′ (≤-trans j′<j j<i) (≤-trans j′<j j<i′))
+
+Glu-wellfounded-≡ : ∀ {i j} (j<i : j < i) →  (λ {A B} → Glu-wellfounded i {j} j<i (𝕌-wellfounded j) {A} {B}) ≡ (_⊢_®[ j ]_)
```

Comment:
this lemma makes sense. I think there could be a more general version, but I think it will come up later.